spec/validation/loader_spec.rb

```diff
@@ -47,9 +47,66 @@
       end.to_h
     end
 
+    let :example_plugin_configure_questions_hash do
+      configure_sections.map do |section|
+        [
+          section, [{
+            identifier: "example_plugin_#{section}_identifier",
+            question: "example_plugin_#{section}_question",
+            dependent: [
+              {
+                identifier: "example_plugin_#{section}_dependent_identifier",
+                question: "example_plugin_#{section}_dependent_question",
+              }
+            ]
+          }]
+        ]
+      end.to_h
+    end
+
     let :filesystem do
```

Comment:
Replace with `begin do FileSystem.root_setup do .... end end` notation. This removes the need to go `FileSystem.test` for each test.

Reply:
Yep, done in https://github.com/alces-software/metalware/commit/691ac8706964c7b5fd0220e30d8e82bcc097d012.